<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartViewDialog</class>
 <widget class="QDialog" name="ChartViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="exportImageButton">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyToClipboardButton">
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ScatterCheckBox">
       <property name="text">
        <string>散点图</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="SplineCheckBox">
       <property name="text">
        <string>曲线拟合</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numOfDuplicatePointsLabel">
       <property name="text">
        <string>重复点个数：0个</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pValueLabel">
       <property name="text">
        <string>p值：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="r2ValueLabel">
       <property name="text">
        <string>r^2值：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DataLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
